index.js

```diff
@@ -138,11 +138,11 @@ sudokuSolver.solve = function (options) {
     solver(problem);
 
     return {
-        toString: function(style) {
+        toString: function () {
             // We just return the stringified solution (styled or not)
             return output (_solution, style, true)
```

Comment:
What do you mean? The `style` variable is defined in line 53. Here it should not exist because it exists already, will never be used at this points and overwrites the original `style` variable.